tests/uts/test_agent.py

```diff
@@ -85,7 +104,7 @@ def run_cmd_on_agent(self, cmd, job_id=None):
     def test_signal_existed_process(self):
         job_id = self.run_cmd_on_agent(['bash', '-ce', 'sleep 5'])
         job_id = self.run_cmd_on_agent('signal', job_id)
```

Comment:
try not to use the same var name... use `job_id` and `signal_job_id` or something